### ISSUE TITLE
task feign death

### ISSUE THREAD

**Issue by liuboflyon145** [zcaudate/cronj#19]
my project used cronj for auto task,but have an issue it is that the task feign death , none log to print, what's happen with it?

**Comment by hoebat** [comment]
Can you give an example?

**Comment by liuboflyon145** [comment]
![image](https://user-images.githubusercontent.com/6317950/33113339-e5bd4b7a-cf92-11e7-966f-b9f86a9de880.png)
this is my application run log image. the task running has print execute log, but after 2017-11-22 01:20:00 the task did not print execute log, so that my application didn't normally running.


{:id "handle-deferred-payment-interest-task"
   :handler exec-fn
   :schedule "30 30 3 * * * *"
   :pre-hook  exec-log-fn
   :opts {:output "start handle-deferred-payment-interest-task"}}

**Comment by hoebat** [comment]
How long was the task running for before it stopped?

**Comment by liuboflyon145** [comment]
a task per ten seconds run, other task exec only once every day . all task running 2 days on line.

**Comment by hoebat** [comment]
I’m actually not sure how I can help you with this. If other tasks are running besides this one, it seems that there is a problem with the task. Have you tried using ‘simulate’ to try reproduce the error?

**Comment by liuboflyon145** [comment]
What I'm trying to say is that after 1.20 a.m., all the tasks are not done. I restarted the server and got back to normal. I'm afraid this problem still exists. But do not know how to troubleshoot, reproduce the problem.

**Comment by hoebat** [comment]
It might be that one of the thread controlling the clock died. However, this is pretty unlikely and from the information you gave, it’s not clear what is happening 
	- did you check memory usage?
	- did you check if the rest of the system is working?
	- have you got monitoring besides logs?

For testing/reproducing Try this.
	- http://docs.caudate.me/cronj/#simulation-testing <http://docs.caudate.me/cronj/#simulation-testing>
